### ISSUE TITLE
import of aiida_wannier90 triggers load_profile

### ISSUE THREAD

**Issue by ltalirz** [aiidateam/aiida-wannier90#36]
Noticed this on the aiida-registry - importing the module fails because it tries playing around with the profile configuration

https://travis-ci.org/ltalirz/aiida-registry/builds/590118189#L521-L544

Other plugins do not do this.

**Comment by greschd** [comment]
Seems to already be fixed in the ``develop`` branch -- the issue was importing from ``aiida.common.utils``, which in turn tried configuring the logging.

I'm not sure what a good strategy for knowing if problems are already (being) fixed from the side of the registry.

**Comment by ltalirz** [comment]
> Seems to already be fixed in the develop branch -- the issue was importing from aiida.common.utils, which in turn tried configuring the logging.

Thanks, good to know!

> I'm not sure what a good strategy for knowing if problems are already (being) fixed from the side of the registry.

For the moment I've made "installing & importing" a separate travis job that is allowed to fail (and I'll keep an eye on this in upcoming PRs).
I'd suggest you keep this issue open until the next release is made into `master`.

**Comment by greschd** [comment]
The travis job takes the packages from pypi, right? It might make sense to also have one that tries installing the current git master.

**Comment by ltalirz** [comment]
> The travis job takes the packages from pypi, right?

yes, well - it takes the packages from the "pip_url", which can either be a link to the repo (git+https://...) or the name of a registered package.
This is also what will be shown as the installation instruction for a plugin when clicking on the details page of a plugin (once this PR is merged https://github.com/aiidateam/aiida-registry/pull/80).

I guess the assumption is that stuff released into `master` will be made available on pypi as well.

We can (and probably will) add more installs in the future.
Besides the "master" branch, there are actually a number of plugins that have a 'pre-release' on pypi, which is not released in to `master` and will not be picked up by pip unless we specify `pip install --pre`.
For the moment I thought it is better to keep things simple.

**Comment by greschd** [comment]
Just noticed this is a duplicate of #13.
